### DISCUSSION TITLE
增值税发票自动识别 | YuZhangWang的领域

### DISCUSSION THREAD

**Discussion by giscus** [YuZhangWang/YuZhangWang.github.io#18] [Announcements]
# 增值税发票自动识别 | YuZhangWang的领域

绪论课题的研究背景及意义增值税专用发票是商业的行政单据[1]，由于实现凭开具收据对购进费用扣税，而进货方要向商品销售方交纳所得税，所以它是完税单据，同时具备对商品销售方的税法义务和购物方进款税金的法律证实的意义，是有效所得税普征性和公平性的体现。因此，增值税发票的两大功能为交易凭据功能和缴税凭据功能。伴随着正在迅发展的科技水平和我国经济情况，在普通的日常生活中，增值税发票的投入使用，成为了不可避免

https://yuzhang.wang/139-invoice-identification/

**Comment by imjiaoyuan** [comment]
学到了~

**Reply by YuZhangWang** [reply]
Thanks for watching, but this project is not worth learning. The work is relatively rough, and the two technologies used are relatively old.
